index.js.flow

```diff
@@ -393,10 +391,10 @@ type IndexOptions = {|
   partialFilterExpression?: { [string]: mixed, ... },
   collation?: CollationParam,
   session?: ClientSession,
-|};
+|}>;
 
 type AggregationPipeline = mixed[];
```

Comment:
This should be reasonably typeable, at the stage name level at least. E.g.

```
type AggregationStageType = '$match' | '$project' | '$limit' ...

type AggregationStage = { [AggregationStageType]: mixed }

type AggregationPipeline = AggregationStage[];
```

Reply:
06d000acc4dc8c1b54462f41a787821a53f57048